src/Components/AccordionsContainer/AccordionsContainer.tsx

```diff
@@ -11,8 +13,36 @@ type Props = {
   submitted: number;
 };
 
+// transforms config benefits and returns array of accordion object array
+const formatCategoryBenefits = (categoryBenefits: Config['category_benefits']): BenefitAccordion[] => {
+  return Object.entries(categoryBenefits).map(([categoryKey, categoryValue]) => {
+    const categoryName: FormattedMessageType = categoryValue.category_name;
+    const categoryOptions: CategoryOptions = Object.entries(categoryValue.benefits).reduce(
+      (options, [benefitKey, benefitValue]) => {
+        const benefitMessage: FormattedMessageType = (benefitValue as Record<string, FormattedMessageType>).message;
+        options[benefitKey] = benefitMessage;
+        return options;
+      },
+      {} as CategoryOptions,
+    );
+
+    return {
+      categoryName,
+      categoryOptions,
+    };
+  });
+};
+
 const AccordionsContainer = ({ componentDetails, submitted }: Props) => {
-  const { formData } = useContext(Context);
+  const { formData, config } = useContext(Context);
+  let formattedBenefits: BenefitAccordion[] = [];
+
+  if (config) {
```

Comment:
There seems to be a lot of checking if config is defined. Because of this, I think that we should create a config hook to check if the config is defined.

For example:
```ts
function useConfig(name: string) {
  const { config } = useContext(Context);

  if (config === undefined) {
    throw new Error('config is not defined');
  }
  
  if (config[name] === undefined) {
    throw new Error(`'${name}' does not exist in the config`);
  }

  return config[name];
}
```

Usage
```ts
const Component = () => {
  // Should be type safe
  const zipcodes = useConfig('zipcodes');

  ...
}
```